src/main/java/org/damap/base/rest/DmpDocumentResource.java

```diff
@@ -41,34 +43,74 @@ public class DmpDocumentResource {
    */
   @GET
   @Path("/{dmpId}")
+  @Deprecated(since = "4.3.0")
   public Response exportTemplate(
       @PathParam("dmpId") long dmpId, @QueryParam("template") ETemplateType template) {
     log.info("Return DMP document file for DMP with id=" + dmpId);
 
+    return this.export(dmpId, template, true, "docx");
+  }
+
+  @GET
+  @Path("/{dmpId}/export")
+  @Produces(MediaType.APPLICATION_OCTET_STREAM)
+  public Response export(
+      @PathParam("dmpId") long dmpId,
+      @QueryParam("template") ETemplateType template,
+      @QueryParam("download") Boolean download,
```

Comment:
By specifying a default value, we may be able to get rid of the null check later.
```suggestion
      @QueryParam("download") @DefaultValue("true") boolean download,
```

Reply:
Done